src/main/kotlin/blackJack/domain/TotalScore.kt

```diff
@@ -2,18 +2,22 @@ package blackJack.domain
 
 object TotalScore {
     fun getScore(cards: List<Int>): Int {
```

Comment:
이왕 만든 `Card`를 사용하면 어떨까요?

```suggestion
    fun getScore(cards: List<Card>): Int {
```